public/index.js

```diff
@@ -139,14 +148,42 @@
     element.style.backgroundPosition = parseInt(value) * illustrationSize + "px";
   }
 
-  function addOnInputToElement(element, func){
-    element.oninput = function(){ func() }
-  }
-
   function emojiSprite(){
     var value = document.getElementsByClassName("range")[0].value;
     var element = document.getElementsByClassName("friends__emoji-sprite")[0];
     changeBackgroundPosition(element, value, -180)
   }
 
+  function emailValidator(emailAddress){
+    var regex = RegExp(
+      '^(([^<>()\\[\\]\\\\.,;:\\s@"]+(\\.[^<>()\\[\\]\\\\.,;:\\s@"]+)*)|(".' +
```

Comment:
Yea I copied it. It looks odd because I wanted to split it over multiple lines because it was too long I did this by turning it into a string. As it is a string I had to then cancel out the backslashed with backslashes. I found it here : http://stackoverflow.com/questions/46155/validate-email-address-in-javascript

I thought 2786 upvotes probably meant it was safe to use.